content/linear_algebra/README.md

```diff
@@ -3,6 +3,11 @@
 Incharge: Sathiya
```

Comment:
Can you add your full name? This will help students (and staff) find you more easily.